### PR TITLE
TE-351 Add placeholder screen for mobile view in production

### PR DESCRIPTION
**Pull request by marcdavi-es**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-351)

### What **one** thing does this PR do?
Adds placeholder screen for mobile view in production

screen shot 2018-04-27 at 17 57 25

### CODE REVIEW

File: styleguide/styleguide-components/MobilePlaceholder/component.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      isolate: false,
+      color: color.white,
+      textDecoration: 'none',
+    }
+  }
+})
+
+class Markup extends React.Component {
+
+  state = {
```
Comment (Jonnygwi) [inline styleguide/styleguide-components/MobilePlaceholder/component.js]: Do you need to do this if you set a defaultValue to the input?

Comment (marcdavi-es) [reply styleguide/styleguide-components/MobilePlaceholder/component.js]: I prefer to be explicit about the state which we intend the component to hold.

Comment (davidpelayo) [reply styleguide/styleguide-components/MobilePlaceholder/component.js]: Totally agree with @marcdavi-es

File: styleguide/styleguide-components/StyleGuideRenderer/getIsDevelopmentServer.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const getIsDevelopmentServer = () => !!webpackHotUpdate;
```
Comment (davidpelayo) [inline styleguide/styleguide-components/StyleGuideRenderer/getIsDevelopmentServer.js]: I would prefer to inject environment variables (via webpack) instead, but this should work fine.

**Pull request merged**